### ISSUE TITLE
stackByTable("..zip") can't find DP code

### ISSUE THREAD

**Issue by annakat** [NEONScience/NEON-utilities#111]
Just downloaded the 'Plant presence and percent cover data' DP1.10058.001, release 2021 from the data portal stackByTable gives me the following error. 

`stackByTable(filepath="./orig_data/NEON_presence-cover-plant.zip", saveUnzippedFiles = F)`

```
Error in stackByTable(filepath = "./orig_data/NEON_presence-cover-plant.zip",  : 
  DP0.10058.001 is not a properly formatted data product ID. The correct format is DP#.#####.001, where the first placeholder must be between 1 and 4.
```

Unzipping the data locally and `stackByTable(filepath="./orig_data/NEON_presence-cover-plant/", saveUnzippedFiles = F)` works just fine.

**Comment by cklunch** [comment]
@annakat Yep, this is a known issue, described here: https://www.neonscience.org/impact/observatory-blog/known-issues-neonutilities-200
Sorry for the bug! It's fixed in the development version of neonUtilities 2.0.1 here on GitHub, but you can also work around it for now, like you found. I'll post again here when 2.0.1 goes to CRAN.

**Comment by cklunch** [comment]
This is resolved in neonUtilities 2.0.1, on CRAN now.
